prime-router/src/testIntegration/resources/datatests/FHIR_to_HL7/sample_SR_1_20230302-0001.hl7

```diff
@@ -1,10 +1,10 @@
-MSH|^~\&|CDC PRIME - Atlanta^2.16.840.1.114222.4.1.237821^ISO|Simple Report^CDPH000085^CLIA|CDPH CA REDIE^2.16.840.1.114222.4.3.3.10.1.1^ISO|CDPH_CID^2.16.840.1.114222.4.1.214104^ISO|20230330123236+0000||ORU^R01^ORU_R01|1519ae86-98f4-40d8-9780-d5211186e5b5|P|2.5.1|||||USA|UNICODE UTF-8|||PHLabReport-NoAck^ELR_Receiver^2.16.840.1.113883.9.11^ISO
-SFT|Centers for Disease Control and Prevention|2c5a603|PRIME ReportStream|2c5a603||20230329192441+0000
+MSH|^~\&|CDC PRIME - Atlanta^2.16.840.1.114222.4.1.237821^ISO|Simple Report^CDPH000085^CLIA|CDPH CA REDIE^2.16.840.1.114222.4.3.3.10.1.1^ISO|CDPH_CID^2.16.840.1.114222.4.1.214104^ISO|20230330053236-0700||ORU^R01^ORU_R01|1519ae86-98f4-40d8-9780-d5211186e5b5|P|2.5.1|||||USA|UNICODE UTF-8|||PHLabReport-NoAck^ELR_Receiver^2.16.840.1.113883.9.11^ISO
```

Comment:
Did this change because it's now using the receiver settings?

Reply:
Yea, basically in a recent PR, I [changed the translation tests](https://github.com/CDCgov/prime-reportstream/pull/12220/files#diff-1ec4eea9e9de183f19278f6af6098fb16097e8467cdd11c01e3bcd24ffe5ea12) to have a hl7 translation config, and that resulted in these timezones changing. Since I was able to bring the receiver settings over in this PR, I was able to change these datetimes back to their prior values.